### DISCUSSION TITLE
SmartAutoClicker-2.1.4

### DISCUSSION THREAD

**Discussion by Nain57** [Nain57/Smart-AutoClicker#283] [Announcements]
# Bug fixes
* Fixed a crash occurring when using the debug overlay in a scenario that can change the screen orientation
* Fixed the validity check of the condition selector, this was leading to the possible capture of the area outside the screenshot, which was causing a crash.
* Various stability improvements.

# Note about Smart AutoClicker copies
I have found several copies of Smart Autoclicker on various platforms (that's what can happen when you are open source). As far as I know, those applications are not open source. This mean you can't really know what the application is doing, and with the required permissions for image detection, it can easily collect everything displayed on your screen and inject actions without you noticing.

I do NOT recommend to use those apps, they are a dangerous threat to your phone's security. 

Here are the copies I have found:
* by AppMatePlus: https://play.google.com/store/apps/details?id=com.gameclicker.autoclicker.pro
* by feng_quick_dev: https://play.google.com/store/apps/details?id=autoclick.ai.bot.pro
* by KKVGAME: https://ultra-autoclicker.fr.softonic.com/android

If you are the developer of one of those applications and can provide the open source code of your app, I will remove it from this malicious software list.

# Links
[F-Droid](https://f-droid.org/packages/com.buzbuz.smartautoclicker/)
[PlayStore](https://play.google.com/store/apps/details?id=com.buzbuz.smartautoclicker)

<hr /><em>This discussion was created from the release <a href='https://github.com/Nain57/Smart-AutoClicker/releases/tag/2.1.4'>SmartAutoClicker-2.1.4</a>.</em>

**Comment by thefancynerd** [comment]
Hey @Nain57 :) is it possible perchance, while using on Android since there seems to be a mouse restriction on some games sadly but the cursor shows up on screen just the game disables tap. I was wondering if you could possibly make it so that the application instead of buttons being placed actually the button follows your mouse cursor, when you are on your target the button is activated by the click. This would be a wonderful backdoor on how to achieve this. 

maybe this is something you would be willing to look into? i would love to talk about it, test maybe i have a phone that could be used for whatever. I love the work being done on this so this would be groundbreaking to achieve. especially if it follows the mouse in realtime and clicks as described.

**Comment by Mohmotte** [comment]
Click are more slower for me...

**Comment by AnTijojo** [comment]
The first thing I'd like to tell you is that there are not enough additional settings. More hypky so that there are more options.

Example: 1
I'm playing and the character is spinning around me. It is inconvenient to make 10 sknin shots, so why not do pixel by pixel?

Example: 2
I make a screenshot of the game to enlarge the object in order to fix it and then beat or other actions, but so does not get. Bonaly the screen resolution changes.

Example: 3, 4
Let's say I play and see the inscription of the character, I fexiruyu it but the background does not allow you to beat "because it is constantly different" why not make it so that bfla function "match"? From 1% to 100%. That gives me two reasons! The inscription he sees, but you need to hit gorazhno lower and as the character rotates around us need to cardinatami it can not do. Make it necessary to hit a couple of a few pixels below. Let's say to make an arrow or something like that to be clear where he will hit when changing settings.

Thank you for reading and make a good update!

I translated all my thoughts through a translator. If there are any mistakes, I apologize.

**Comment by ZK-Khann** [comment]
Hello 
I really like this app and i am also a software developer but I am quite new. I tried contacting the owner maybe he does not respond to emails. I wanted to ask if a feature could be implemented that makes the play and pause button go away if chosen like from the menu an option as I need it and if the owner is interested let me know as its a feature i want to use in something bigger 
Thanks

**Comment by Nain57** [comment]
https://github.com/Nain57/Smart-AutoClicker/discussions/304

**Comment by ZK-Khann** [comment]
Hello
Wanted to ask about a feature being implemented .
Thanks
